docker/package/defaults/tezos-node.conf

```diff
@@ -4,6 +4,7 @@
 # SPDX-License-Identifier: MPL-2.0
 
 # shellcheck disable=SC2034
+DATA_DIR="/var/lib/tezos/node"
```

Comment:
Btw, are these things set only for a local scope and unset when this "local scope" is exited? Imagine I run another application that uses `DATA_DIR` env variable. Is there a chance that it will see `DATA_DIR="/var/lib/tezos/node"`?

Reply:
It's environment only within systemd service executable, e.g. in scope of `tezos-node-start`.